modules/claims_api/app/models/claims_api/evss_claim.rb

```diff
@@ -56,5 +56,44 @@ def supporting_documents
         []
       end
     end
+
+    def self.services_are_healthy?
+      # TODO: we should add check for Okta and SAML Proxies being up as well
+      last_mvi_outage = Breakers::Outage.find_latest(service: MVI::Configuration.instance.breakers_service)
+      mvi_up = (last_mvi_outage.blank? || last_mvi_outage.end_time.present?)
+
+      last_evss_claims_outage = Breakers::Outage.find_latest(service: EVSS::ClaimsService.breakers_service)
+      evss_claims_up = last_evss_claims_outage.blank? || last_evss_claims_outage.end_time.present?
+
+      last_evss_common_outage = Breakers::Outage.find_latest(service: EVSS::CommonService.breakers_service)
+      evss_common_up = last_evss_common_outage.blank? || last_evss_common_outage.end_time.present?
+      mvi_up && evss_claims_up && evss_common_up
+    end
+
+    def self.healthy_service_response
+      {
+        data:  {
+          id: 'claims_healthcheck',
+          type: 'claims_healthcheck',
+          attributes: {
+            healthy: true,
+            date: Time.zone.now.to_formatted_s(:iso8601)
+          }
+        }
+      }.to_json
+    end
+
+    def self.unhealthy_service_response
+      {
+        errors: [
+          {
+            title: 'ClaimsAPI Unavailable',
+            detail: 'ClaimsAPI is currently unavailable.',
+            code: '503',
+            status: '503'
+          }
+        ]
+      }.to_json
+    end
```

Comment:
The `healthy_service_response` and `unhealthy_service_response` methods feel out of place in this model. They are specific to the needs of the `healthcheck` controller action. They don't have any fundamental relationship to anything else in the model(s) and aren't theoretically re-usable. Moving them to the controller or a serializer seems more appropriate to me.

Reply:
I commented on the wrong item, but essentially these were moved out into the controller.